### PR TITLE
feat(dev): set path to geoip location database

### PR DESCRIPTION
**Pull request by calavera**
This will allow testing edge handlers to geoip location capabilities.

That flag was introduced in https://github.com/netlify/traffic-mesh/pull/1199/files

Signed-off-by: David Calavera <david.calavera@gmail.com>

### CODE REVIEW

**Review by erezrokah — CHANGES_REQUESTED** [review]
This looks good.
Does it need a version bump for the npm traffic mesh agent package?
I'm getting:
```
error: Found argument '--location-db' which wasn't expected, or isn't valid in this context
```
When passing the argument

**Comment by calavera** [comment]
That's weird, I assumed that the most recent version had that option already, I'll double check.

File: package-lock.json [hunk 1]
```diff
@@ -10111,7 +10111,7 @@
         },
         "pify": {
           "version": "2.3.0",
-          "resolved": "https://registry.npmjs.org/pify/-/pify-2.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/pify/-/pify-2.3.0.tgz",
```
Comment (erezrokah) [inline package-lock.json:10114]: Going to merge this once I figure out these `https->http` changes

**Review by erezrokah — CHANGES_REQUESTED** [review]
~~@calavera I'm still getting the same error with version `0.24.0`~~
~~Maybe it's because I'm submitting a wrong db?~~
Argument name should be `geo`

**Pull request merged**